packages/devtools_app/lib/src/shared/console/eval/eval_service.dart

```diff
@@ -108,4 +110,25 @@ class EvalService extends DisposableController with AutoDisposeControllerMixin {
       disableBreakpoints: true,
     );
   }
+
+  Future<InstanceRef?> findObject(
+    AdaptedHeapObject object,
+    IsolateRef isolateRef,
+  ) async {
+    final isolateId = isolateRef.id!;
+
+    final theClass = (await serviceManager.service!.getClassList(isolateId))
+        .classes!
+        .firstWhere((ref) => object.heapClass.matches(ref));
+
+    final instances = await serviceManager.service!.getInstances(
+      isolateId,
+      theClass.id!,
+      preferences.memory.refLimit.value,
+    );
+
+    return (instances.instances ?? []).firstWhereOrNull(
+      (i) => i is InstanceRef && i.identityHashCode == object.code,
+    ) as InstanceRef?;
```

Comment:
I think you can remove this cast if you explicitly define the type parameter for the empty list as `InstanceRef`.

Reply:
nope, because instances returns list of ObjRef